main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ParentStationValidator.java

```diff
@@ -31,17 +34,20 @@
 import org.mobilitydata.gtfsvalidator.table.GtfsStopTimeSchema;
 
 /**
- * Validates `location_type` of the referenced `parent_station`.
+ * Validates parent stations are used and have the correct `location_type`.
  *
- * <p>Generated notice: {@link WrongParentLocationTypeNotice}.
+ * <p>Generated notices:
+ *
+ * <ul>
+ *   <li>{@link WrongParentLocationTypeNotice}.
+ *   <li>{@link UnusedParentStationNotice}.
  */
```

Comment:
The `aggregateJavaDoc` it's failing due to a missing closing tag,
```suggestion
 * <ul>
 *   <li>{@link WrongParentLocationTypeNotice}.
 *   <li>{@link UnusedParentStationNotice}.
 * </ul>
 */
```